doc/releases/migration-guide-3.7.rst

```diff
@@ -247,6 +247,9 @@ Bluetooth HCI
  * As part of the new HCI driver API, the ``zephyr,bt-hci-ipc`` chosen property is only used for
    the controller side, whereas the HCI driver now relies on nodes with the compatible string
    ``zephyr,bt-hci-ipc``.
+ * The ``BT_NO_DRIVER`` Kconfig option was removed. HCI drivers are no-longer behind a Kconfig
+   choice, rather they can now be enabled and disabled independently, mostly based on their
+   respective devicetree node being enabled or not.
```

Comment:
please indent all the bullet points following "* A new HCI driver API was ...." so that they effectively fall under it and it's clear they're all related to 72323 (if I understood correctly?)
Don't forget blank line before the new "sub list" :)

Reply:
@kartben done!